--- src/kim_publication.desktop.orig	2009-02-10 11:14:08.000000000 +0200
+++ src/kim_publication.desktop		2010-01-12 01:36:45.503348819 +0200
@@ -19,21 +19,20 @@
 [Desktop Entry]
 Type=Service
 ServiceTypes=KonqPopupMenu/Plugin,image/*
-Actions=Rename;SortByDate;SendByMail;_SEPARATOR_;PeleMele;Slideshow;Galery;_SEPARATOR_;Watermark;Border;_SEPARATOR_;Animation;Multiburst;Adjoin;_SEPARATOR_;Apropos;
-Encoding=UTF-8
+Actions=Rename;SortByDate;SendByMail;PeleMele;Slideshow;Galery;Watermark;Border;Animation;Multiburst;Adjoin;Apropos;
 X-KDE-Submenu=Kim - Treatment and publication
 X-KDE-Submenu[fr]=Kim - Traitement et publication
 
 [Desktop Action Slideshow]
 Name=Create a Flash slideshow
 Name[fr]=Créer un slideshow en Flash
-Icon=html
+Icon=text-html
 Exec=kim_slideshow %U
 
 [Desktop Action Galery]
 Name=Create a html gallery
 Name[fr]=Créer un album en html
-Icon=html
+Icon=text-html
 Exec=kim_galery %U
 
 [Desktop Action PeleMele]
@@ -45,25 +44,25 @@
 [Desktop Action SendByMail]
 Name=Send by mail
 Name[fr]=Envoyer par mail
-Icon=kmail
+Icon=internet-mail
 Exec=kim_resizeandsend %U
 
 [Desktop Action SortByDate]
 Name=Sort by date
 Name[fr]=Ordonner par date
-Icon=today
+Icon=view-sort-descending
 Exec=kim_sortbydate %D %U
 
 [Desktop Action Rename]
 Name=Rename images
 Name[fr]=Renommer des images
-Icon=run
+Icon=edit-rename
 Exec=kim_rename %D %U
 
 [Desktop Action Adjoin]
 Name=Create a multi-files Tiff image
 Name[fr]=Créer une image Tiff multi-pages
-Icon=attach
+Icon=mail-attachment
 Exec=convert -adjoin %U %D/my_images.tiff
 
 [Desktop Action Animation]
@@ -99,6 +98,6 @@
 [Desktop Action Apropos]
 Name=About Kim
 Name[fr]=A propos de Kim
-Icon=help
+Icon=help-about
 Exec=kim_other about %U
 
